client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -265,6 +301,7 @@ protected byte getWriteVersion() {
         protected void declareVersions() {
             version(0).revision(0, this::write00, this::read00);
             version(0).revision(1, this::write01, this::read01);
+            version(1).revision(0, this::write10, this::read10);
```

Comment:
Please increase the version number only if required. Also, add tests to verify the versioning compatability.

Reply:
The version number is not increased. Instead there's a new revision, the test for this new revision's version compatibility has been added to `ReaderGroupConfigTest`.

Reply:
increasing the revision is the right thing to do.